### DISCUSSION TITLE
טעות העתקה בחלק בגליון תלכ"צ2 תרגיל 1 סעיף e'

### DISCUSSION THREAD

**Discussion by micha-sherman** [NuclearGandhi/technion_second_brain#9] [טעויות ותיקונים]
אני חושב שבטעות השתמשת בft של המהירות המשתנה ולא בזה של הקבועה עבור החישוב הזמן עבור מהירות זוותית קבועה

**Comment by NuclearGandhi** [comment]
השתמשתי ב-$`f`$ של סעיף c. או שאולי התכוונת למשהו אחר?

**Reply by micha-sherman** [reply]
![image](https://github.com/user-attachments/assets/c2b48fc9-c93b-459f-aecf-d2c66f759212)
![image](https://github.com/user-attachments/assets/22b9311a-a0c4-43ee-ba2c-458a1dd87d4d)
עבור מהירות סיבוב קבועה זה f=0.4.

**Reply by NuclearGandhi** [reply]
נכון, אבל אז בסעיף 3 מצאנו f מקסימלי בגלל הדרישה על טיב פני השטח, בלי קשר אם המהירות סיבוב קבועה או לא. אם הייתי משתמש במה שנתון, זה לא היה עונה על דרישות טיב פני השטח.

**Reply by micha-sherman** [reply]
אתה צודק, מבאס שזאת שאלה בהבנת הנקרא
